core/steps/network/interface_create.go

```diff
@@ -68,10 +69,19 @@ func (s *createInterface) Do(ctx context.Context) ([]planner.Procedure, error) {
 	}
 
 	if s.status == nil {
-		s.status = &models.NetworkInterfaceStatus{}
```

Comment:
That's not the way how pointers work.

Reply:
And we create the status if its not found:

https://github.com/weaveworks/flintlock/blob/main/core/plans/microvm_create_update.go#L143

Reply:
The code must've been hung over from a previous life.